apps/trading/client-pages/referrals/referral-statistics.tsx

```diff
@@ -229,11 +270,16 @@ export const Statistics = ({
 
   const currentBenefitTierTile = (
     <StatTile title={t('Current tier')}>
-      {currentBenefitTierValue?.tier || 'None'}
+      {currentBenefitTierValue?.tier || benefitTiers[0]?.tier || 'None'}
```

Comment:
suggestion: shouldn't we show the "worst" tier only for the preview, but leave "None" for the actual referee stats.

Reply:
Thanks. I pushed a change where I'm checking if it is a preview by verifying if the referee object is null.